api/python/provisioner/commands/auto_deploy.py

```diff
@@ -60,13 +59,14 @@ def deployment_validations(deploy_check):
             check_res = check_cmd().run(deploy_check)
         except Exception as exc:
             raise ValueError("Error During Deployment "
-                            f"{deploy_check} Validations: {str(exc)}")
+                             f"{deploy_check} Validations: {str(exc)}")
         else:
             if "pre" in deploy_check:
```

Comment:
I think it is better to check with `deploy_check.startswith("pre-")`. It may be more accurate.

Reply:
DEPLOY_PRE_CHECKS = "deploy_pre_checks"    DEPLOY_POST_CHECKS = "deploy_post_checks"
We have pre and post in middle so we cannot use startswith here

Reply:
Ok, then we can update
we can update
```python
if "pre" in deploy_check:
```
by the 
```python
if deploy_check in (GroupChecks.DEPLOY_PRE_CHECKS, DEPLOY_PRE_CHECKS.UNBOXING_PRE_CHECKS)
```

Reply:
comment is addressed